cpu/samd21/periph/i2c.c

```diff
@@ -197,10 +202,10 @@ int i2c_init_master(i2c_t dev, i2c_speed_t speed)
     i2c_poweron(dev);
 
     /* Start timeout if bus state is unknown. */
-    while (!(I2CSercom->STATUS.reg & SERCOM_I2CM_STATUS_BUSSTATE(1))) {
+    while ((I2CSercom->STATUS.reg & SERCOM_I2CM_STATUS_BUSSTATE_Msk) == BUSSTATE_UNKNOWN) {
```

Comment:
The define above defines `BUSSTATE_UNKNOWN` as `SERCOM_I2CM_STATUS_BUSSTATE(0)`, however this was `SERCOM_I2CM_STATUS_BUSSTATE(1)` before. Is this correct?